multinetjs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "multinet",
-  "version": "0.5.0",
+  "version": "0.6.0",
```

Comment:
Usually I like to defer the multinetjs version number bump until the moment I actually release it to NPM, but you can leave this instance of it intact.